js/apps/system/_admin/aardvark/APP/frontend/js/views/dashboardView.js

```diff
@@ -1157,18 +1157,22 @@
     template: templateEngine.createTemplate('dashboardView.ejs'),
 
     checkEnabledStatistics: function () {
-      if (!frontendConfig.statisticsEnabled || frontendConfig.db !== '_system') {
+      if (frontendConfig.statisticsEnabled && frontendConfig.db !== '_system') {
         $(this.el).html('');
         if (this.server) {
           $(this.el).append(
-            '<div style="color: red">Server statistics (' + this.server + ') are disabled.</div>'
+            '<div style="color: red">Server statistics for target (' + this.server + ') are disabled in this database. Log into "_system" database to show statistics.</div>'
```

Comment:
I prefer the term "node" instead of "target". In the left hand menu of the web UI we also use the term "Nodes".
```suggestion
            '<div style="color: red">Server statistics for node (' + this.server + ') are disabled in this database. Log into "_system" database to show statistics.</div>'
```